indico/modules/events/editing/blueprint.py

```diff
@@ -62,12 +62,20 @@
 _bp.add_url_rule('/editing/api/service/disconnect', 'api_service_disconnect', service.RHDisconnectService,
                  methods=('POST',))
 _bp.add_url_rule('/editing/api/service/status', 'api_service_status', service.RHServiceStatus)
+_bp.add_url_rule('/editing/<any(paper,slides,poster):type>/editables/files.zip', 'download_archive',
+                 editable_list.RHDownloadArchive)
```

Comment:
I don't think a GET request will work well for this unless you select only few editables. It will also result in a really ugly URL. What about my suggestion of storing the IDs in redis with a token (with a POST request), and then returning only the URL so the client-side code can simply redirect to the download URL? I.e. similar to what we do with the `badge_cache` when printing badges.

Reply:
Why not something like `RHContributionsMaterialPackage`? I would just have to use `POST` in this case and let the frontend handle it properly.

Reply:
But there we use classic forms that just do a normal submit, while editing is all in react... So when doing it like in `RHContributionsMaterialPackage` the client code would need to get the binary data from the AJAX request and use the Blob API to start a download... vs just redirecting to the url starting the download. And from what I can see `URL.createObjectURL()` still doesn't let you specify a file name shown in the download prompt :/